app/marketing/views.py

```diff
@@ -587,18 +587,24 @@ def account_settings(request):
             response['Content-Disposition'] = f'attachment; filename="{name}.csv"'
 
             writer = csv.writer(response)
-            writer.writerow(['id', 'date', 'From', 'To', 'Type', 'Value In USD', 'url'])
+            writer.writerow(['id', 'date', 'From', 'From Location', 'To', 'To Location', 'Type', 'Value In USD', 'url', 'txid', 'token_name', 'token_value'])
             profile = request.user.profile
             earnings = profile.earnings if export_type == 'earnings' else profile.sent_earnings
-            earnings = earnings.all().order_by('-created_on')
+            earnings = earnings.filter(network='mainnet').order_by('-created_on')
```

Comment:
shouldn't this be dynamic to work on local also ?

Reply:
nah it's just for export -> should be fine